content/chainguard/chainguard-enforce/chainctl-docs/how-to-install-chainctl.md

```diff
@@ -2,8 +2,8 @@
 title: "How to Install chainctl for Chainguard Enforce"
```

Comment:
Does this make sense in the `chainctl` directory, or should I pull this doc out to the Chainguard Enforce for Kubernetes directory?

Reply:
I think moving it out of the `chainctl-docs` directory is a good idea. Once the tools are in place to automate publishing `chainctl` docs, I'm thinking anything in there is liable to get clobbered on an update. That, and I'd like to also figure out how to move the entire directory to live under `content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes`

Reply:
(forgot to say, I don't mean to block you from merging this, since the page can get moved at any time)

Reply:
I'm moving it